cellpose_napari/_sample_data.py

```diff
@@ -32,9 +30,13 @@ def _load_cellpose_data(image_name, dname):
 @napari_hook_implementation
 def napari_provide_sample_data():
     from functools import partial
+
+    # Import when users activate plugin
+    from cellpose.utils import download_url_to_file
```

Comment:
this actually will get called when napari starts... but it's not immediately clear where `download_url_to_file` and `imread` area actually used here... might be leftover from previous dev?  can you test removing this altogether?

Reply:
`download_url_to_file` and `imread` are called from function `_load_cellpose_data` defined right above the hook. I thought that this could be a bit confusing when you think about the scope. I would move the function inside the hook scope.

Reply:
ah right, sorry I didn't expand the file.  ok, yeah, let's move these imports up into `_load_cellpose_data`.  napari will call this `napari_provide_sample_data` function when the viewer is created to populate the menus... but the partial here that calls `_load_cellpose_data` will only be called when the user actually selects one of these items from the menu.

Reply:
Please check the modification.